cmd/registry/cmd/upload/bulk/protos.go

```diff
@@ -372,7 +372,7 @@ func referencedProtos(protos []string, root string) ([]string, error) {
 		return nil, err
 	}
 	defer os.RemoveAll(tempDir)
-	args := []string{"-o", tempDir + "/proto.pb"}
+	args := []string{"-o", tempDir + "/proto.pb", "--include_imports"}
```

Comment:
Testing revealed that the "--include_imports" flag had been omitted in the original commit (which had relied on iteratively compiling a list of protos until all were included). Subsequent removal of that iteration broke the uploader.